spec/migrations/20250930000666_remove_grub1_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+
+migration '20250930000666_remove_grub1_spec' do
+  scenarios %w[foreman foreman-proxy-content katello] do
+    context 'tftp_dirs' do
+      let(:answers) do
+        { 'foreman_proxy' => {
+          'tftp_root' => '/var/lib/tftboot',
+            'tftp_dirs' => ["/var/lib/tftboot/grub", "/var/lib/tftbot/grub2"],
+        } }
+      end
+
+      it 'remove grub1' do
+        expect(migrated_answers['foreman_proxy']['tftp_dirs']).not_to include("/var/lib/tftbot/grub")
```

Comment:
I don't think this test will ever fail because of typos.
```suggestion
        expect(migrated_answers['foreman_proxy']['tftp_dirs']).not_to include("/var/lib/tftpboot/grub")
```